sceptre/bridge/config/prod/service-linked-roles.yaml

```diff
@@ -0,0 +1,6 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.0/service-linked-roles.yaml
```

Comment:
is there a reason why you are deploying this into the account? It's not security related and I don't think you actually need it.

Reply:
I was following the instructions provided to me at https://sagebionetworks.jira.com/browse/BRIDGE-3185?focusedCommentId=145723 I had assumed that I needed this. Perhaps I misunderstood the instructions?

Reply:
I don't see anything in the dock that references service-linked-roles.  I would suggested removing this template.